src/cc/export/helpers.h

```diff
@@ -90,6 +90,14 @@ struct _name##_table_t { \
 __attribute__((section("maps/perf_array"))) \
 struct _name##_table_t _name
 
+// Identifier for current CPU used in perf_submit and perf_read
+// Prefer use BPF_F_CURRENT_CPU flag, falls back to call helper for older kernel
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 8, 0)
```

Comment:
I see:(
We **would** be able to check `#ifdef BPF_F_CURRENT_CPU`. However we keep our copy of `bpf.h` and always keep it update to the latest, so no way of checking what user's Kernel actually has.....